src/BursarExportPlugin.tsx

```diff
@@ -71,17 +73,19 @@ export default function BursarExportPlugin() {
   }
 
   return (
-    <Pane
-      defaultWidth="fill"
-      footer={footer}
-      id="pane-batch-group-configuration"
-      paneTitle={<FormattedMessage id="ui-plugin-bursar-export.bursarExports.paneTitle" />}
-    >
-      <ConfigurationForm
-        initialValues={initialValues}
-        onSubmit={submitCallback}
-        formApiRef={formApiRef}
-      />
-    </Pane>
+    <TitleManager page={intl.formatMessage({ id: 'ui-plugin-bursar-export.meta.settingsTitle' })}>
```

Comment:
Implementation looks good here. Although I think you should pass `stripes={stripes}` in as is done in the [ui-cal implementation](https://github.com/folio-org/ui-calendar/pull/502).